src/DataTable.js

```diff
@@ -35,7 +42,14 @@ export default function DataTable(props) {
                 left, top, width, height
             }}
         >
-            <h4>Data Preview</h4>   
+            <h4 className="chw-table-title">{title}</h4>
```

Comment:
I like this layout better than the previous one, looking more organized.

Let's fine-tune the interface of Data Table when we become to better know the usage of this table, reflecting the collaborators' analysis process.

Reply:
Sounds good